src/yuzu/configuration/shared_widget.cpp

```diff
@@ -344,6 +387,43 @@ QWidget* Widget::CreateSpinBox(const QString& given_suffix,
     return spinbox;
 }
 
+QWidget* Widget::CreateDoubleSpinBox(const QString& given_suffix,
+                                     std::function<std::string()>& serializer,
+                                     std::function<void()>& restore_func,
+                                     const std::function<void()>& touch) {
+    const auto min_val = std::strtod(setting.MinVal().c_str(), nullptr);
+    const auto max_val = std::strtod(setting.MaxVal().c_str(), nullptr);
+    const auto default_val = std::strtod(setting.ToString().c_str(), nullptr);
+
+    QString suffix =
+        given_suffix == QStringLiteral("") ? DefaultSuffix(this, setting) : given_suffix;
```

Comment:
```suggestion
        given_suffix == QStringLiteral{} ? DefaultSuffix(this, setting) : given_suffix;
```
just {} might be possible as well, you need to try

Reply:
Doesn't work, Clang wants to use QStringLiteral as an identifier